File: 0122-best-time-to-buy-and-sell-stock-ii/0122-best-time-to-buy-and-sell-stock-ii.java
class Solution {
    public int maxProfit(int[] prices) {
        int sum = 0;        
        
        for(int i = 0; i < prices.length - 1; i++)
        {            
            int today = prices[i];
            int tomorrow = prices[i + 1];
            
            if(today < tomorrow)
                sum += tomorrow - today;            
            
        }
        
        return sum;
    }
}

File: Medium/0122-best-time-to-buy-and-sell-stock-ii/0122-best-time-to-buy-and-sell-stock-ii.js
/**
 * @param {number[]} prices
 * @return {number}
 */
var maxProfit = function(prices) {
    let sum = 0;
    
    for(let i = 0; i < prices.length - 1; i++){
        let today = prices[i];
        let tomorrow = prices[i + 1];
        
        if(today < tomorrow){
            sum += tomorrow - today;
        }
    }
    
    return sum;
};